src/main/java/chopchop/logic/edit/EditDescriptor.java

```diff
@@ -0,0 +1,6 @@
+// EditDescriptor.java
+
+package chopchop.logic.edit;
+
+public class EditDescriptor {
+}
```

Comment:
Why is there an empty class?

Reply:
It's just to act as a base class for the 3 sub-edit-descriptors, which was necessary cos of the way i wrote `parseEditCommand`

Reply:
The `editType` field and accessor seems to be shared by all 3 class, maybe consider putting it in the base class?

Reply:
5head